src/js/fc.js

```diff
@@ -798,10 +798,10 @@ const FC = {
     getSupportedSerialRxTypes: () => {
         if (FC.CONFIG.buildOptions?.length) {
             const options = FC.CONFIG.buildOptions;
-            let supportedRxTypes = ['NONE',];
+            let supportedRxTypes = ['NONE'];
```

Comment:
Please rebase. This was already fixed in https://github.com/betaflight/betaflight-configurator/commit/a23f3cfb33018000815291b9b9b505bb32b6d8b9